modules/kernel/src/main/java/org/opencastproject/kernel/security/RemoteUserAndOrganizationFilter.java

```diff
@@ -164,7 +164,7 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
 
           if (!originalUser.hasRole(GLOBAL_ADMIN_ROLE)) {
             // if the original user did not have system privileges, the target user must not gain those, either.
-            for (String systemRole : GLOBAL_SYSTEM_ROLES) {
+            for (String systemRole : GLOBAL_SYSTEM_ROLES) { // This IS WRONG
```

Comment:
If this is wrong… why is this here?

Reply:
Because... I forgot to remove this. Whoops.